### DISCUSSION TITLE
Integrate Okta with Strapi

### DISCUSSION THREAD

**Discussion by erkand-imeri** [strapi/strapi#7442] [Legacy Topics]
Hello all,

I was looking into into integrating Okta SSO with Strapi, i could not find anything in the documentation page.

For instance Directus has ready made integration for Okta: https://docs.directus.io/guides/authentication.html#single-sign-on-sso

Is it possible to integrate Okta with Strapi?

**Comment by derrickmehaffy** [comment]
Hi @erkand-imeri is this related to SSO on the Admin or for user endpoint authentication?

**Comment by erkand-imeri** [comment]
Hi @derrickmehaffy, this is related to SSO on the Admin.

**Reply by derrickmehaffy** [reply]
Hi @erkand-imeri currently it's not possible without forking the `strapi-admin` package but we do have this planned and you will be able to build your own Okta passport.js provider once we release the SSO feature (will only include a handful of default providers but will be easy to build new ones).

Here is the entry on the roadmap: https://portal.productboard.com/strapi/1-public-roadmap/c/61-authentication-providers-sso

Rough ETA on this is by the end of Q3 2020

**Reply by erkand-imeri** [reply]
@derrickmehaffy can you please show me how is it possible implementing with strapi-admin package?

Is there any working example?

**Reply by derrickmehaffy** [reply]
There is no current working example yet, this feature will be next on development after the Draft/Publish AFAIK and it shouldn't take too long for us to develop.

**Comment by erkand-imeri** [comment]
Anyone has any insight into this?

**Comment by alexeysarakuz** [comment]
any updates @derrickmehaffy ?

**Reply by derrickmehaffy** [reply]
Our release plan for the Admin SSO is end of Q4 2020 (late dec) and we will be using passport.js so pretty much any existing passport.js package will work (and there are many for Okta).

We have no started the development yet as we are working on another feature that will be released mid Dec so there is no alpha or beta test as of yet.

**Comment by boker** [comment]
any updates?

**Reply by derrickmehaffy** [reply]
See: https://github.com/strapi/strapi/discussions/7442#discussioncomment-139019

We have started development, but it's quite likely this feature will be EE license only.
https://portal.productboard.com/strapi/1-roadmap/c/61-sso-authentication-providers-for-the-administration-panel
